### DISCUSSION TITLE
Option to hide tags

### DISCUSSION THREAD

**Discussion by relikd** [the-djmaze/snappymail#560] [Ideas]
I am doing a side-by-side comparison with Rainloop and noticed the new tags feature in the email header section. I won't be using this feature and wondered whether it would be nice to add an option to disable it. That would save a little of precious vertical space (which in my case is always empty).

Also: maybe hide the Spam-Score if it is zero?

![screen](https://user-images.githubusercontent.com/1321720/196677465-f155bc84-8d10-4aef-8d9c-f9c447d7887c.jpg)

**Comment by the-djmaze** [comment]
Well, your mail server supports it and so does SnappyMail.
Maybe there's a way to disable it in your mail server?

**Comment by MI-KY** [comment]
Just to add the reference - this is also discussed in #683

**Reply by relikd** [reply]
ahhh, here was my discussion, I've searched the issues but could not find where I started it. In the meantime I created this issue https://github.com/the-djmaze/snappymail/issues/672 (though I havent had the chance to update and see the changes). Thanks for the backref.
